### PR TITLE
[Release-1.5] update certmanager part in the telemetry addon doc

### PR DESCRIPTION
**Pull request by richardwxn**
the api version is updated for recent cert manager, also there might be multiple short name called certificate from different groups, use the full name instead.

### CODE REVIEW

File: content/en/docs/tasks/observability/gateways/index.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ the [Secret Discovery Service](https://www.envoyproxy.io/docs/envoy/latest/confi
 
     {{< text bash >}}
     $ cat <<EOF | kubectl apply -f -
-    apiVersion: certmanager.k8s.io/v1alpha1
+    apiVersion: cert-manager.io/v1alpha2
```
Comment (ericvn) [inline content/en/docs/tasks/observability/gateways/index.md]: There is one more change needed in the `Cleanup` section

Comment (richardwxn) [reply content/en/docs/tasks/observability/gateways/index.md]: done

**Pull request merged**